src/lang/en.json

```
@@ -1052,6 +1052,13 @@
     "Can be found on:": "Can be found on: {0}",
     "The phone number of the recipient in E.164 format.": "The phone number of the recipient in E.164 format.",
     "Either a text sender ID or a phone number in E.164 format if you want to be able to receive replies.":"Either a text sender ID or a phone number in E.164 format if you want to be able to receive replies.",
+    "RabbitMQ Nodes": "RabbitMQ Management Nodes",
+    "rabbitmqNodesDescription": "Enter the URL for the RabbitMQ management nodes including protocol and port. Example: {0}",
+    "rabbitmqNodesRequired": "Please set the nodes for this monitor.",
+    "rabbitmqNodesInvalid": "Please use a complete URL for RabbitMQ nodes.",
```

what do you think about this variation?

```suggestion
    "rabbitmqNodesInvalid": "Please use a fully qualified (starting with 'http') URL for RabbitMQ nodes.",
```